docs/architecture.md

```diff
@@ -1,3 +1,82 @@
-# Network Architecture
+# Overview
 
-![Architecture](./assets/architecture.png)
+The Azure Trusted Research Environment (TRE) consists of multiple components all encapsulated in networks with restricted in- & egress traffic. One network for management components and one network per Workspace. All traffic has to explicit allowed by the Application Gateway of the Firewall. Azure TRE is build around the concept of data in a Workspace must not be able to exfiltrate the Workspace.
+
+![Architecture overview](./assets/archtecture-overview.png)
```

Comment:
I suggest we move as much as possible of the descriptive text out of the image and into the doc instead. Also, let's see if we can align the boxes in the diagram.

Reply:
Removed most of the text as suggested, except descriptions of elements